package.json

```diff
@@ -22,19 +22,22 @@
   "files": [
     "dist",
     "api-specifications",
-    "core"
+    "core",
+    "sn-table-ext"
   ],
   "engines": {
     "node": ">=16"
   },
   "main": "dist/sn-table.js",
+  "module": "core/esm/index.js",
```

Comment:
To fix error: "export 'useId' (imported as 'React') was not found in 'react'" after upgrading" (https://github.com/mui/material-ui/issues/29860)